src/sentry/incidents/subscription_processor.py

```diff
@@ -562,6 +622,106 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
         # before the next one then we might alert twice.
         self.update_alert_rule_stats()
 
+    def get_anomaly_detected(
+        self, anomaly
+    ) -> bool:  # TODO: add the anomaly type once that's added to Sentry
+        """
+        Helper function to determine whether we care about an anomaly based on the
+        anomaly type and the alert rule's threshold type.
+        """
+        # need to define constants for anomaly types
+        anomaly_type = anomaly.get("anomaly", {}).get("anomaly_type")
+
+        if anomaly_type == "none" or anomaly_type == "no_data":
+            return False
+        elif anomaly_type == "anomaly_high":
+            if (
+                self.alert_rule.threshold_type == AlertRuleThresholdType.ABOVE.value
+                or self.alert_rule.threshold_type == AlertRuleThresholdType.ABOVE_AND_BELOW.value
+            ):
+                return True
```

Comment:
Is this logic correct? 🤔 

So - `anomaly_high` only triggers for above/a&b anomalies?
isn't `anomaly_high` just mean we have high confidence that htis is an anomaly? Not that it's 'above' any trheshold?

Reply:
Confirmed with Tillman that the logic is correct.